ee/codegen/src/__test__/node-inputs/node-input-value-pointer-rules/node-input-value-pointer-rule.test.ts

```diff
@@ -43,11 +44,23 @@ describe("NodeInputValuePointerRule", () => {
   });
 
   it("should generate correct AST for NODE_OUTPUT pointer", async () => {
-    vi.spyOn(nodeContext.workflowContext, "getNodeContext").mockReturnValue({
-      nodeClassName: "TestNode",
-      path: ["nodes", "test-node-path"],
-      getNodeOutputNameById: vi.fn().mockReturnValue("my_output"),
-    } as unknown as BaseNodeContext<WorkflowDataNode>);
```

Comment:
We made a change to `NodeContext`'s interface, which caused a bunch of these spies to fail. I argue that the real thing is just as performant, and in some ways simpler, to just use instead